ui/pages/permissions-connect/connect-page/connect-page.tsx

```diff
@@ -190,6 +203,13 @@ export const ConnectPage: React.FC<ConnectPageProps> = ({
         return namespace === KnownCaipNamespace.Eip155;
       },
     );
+
+    supportedRequestedCaipChainIds = Array.from(
+      new Set([
+        ...supportedRequestedCaipChainIds,
+        ...defaultSelectedNetworkList,
+      ]),
+    );
```

Comment:
not sure I follow this, why would we assign to a variable called `supportedRequestedCaipChainIds` a list of non requested networks?

Reply:
I might be misreading it but the way I'm reading it implies that even if you were to request specific chains via `wallet_requestPermissions` you would get all EVM chains now?